packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -2407,12 +2407,23 @@ describe('Checkout', () => {
                 });
             });
 
+            it('should call `trackFormInteraction` with the error information including `errorCode` when it exists', () => {
+                // Arrange
+                error.errorCode = 'OrderError';
+
+                // Act
+                wrapper.vm.handleErrorState(error);
+
+                // Assert
+                expect(trackFormInteractionSpy).toHaveBeenCalledWith({ action: 'error', error: `error_${error.errorCode}-${error.message}` });
+            });
+
             it('should call `trackFormInteraction` with the error information', () => {
```

Comment:
Could we refactor this to make the test name for explicit? 
```should call `trackFormInteraction` without an `errorCode` when it doesn't exist```

And then I'd do an `// Arrange` with `error.errorCode = null` or similar.